tests/test_hooks/test_checkpoint_hook.py

```diff
@@ -632,3 +632,23 @@ def test_with_runner(self, training_type):
             any(re.findall(r'-[\d\w]{8}\.pth', file) for file in ckpt_files))
 
         self.clear_work_dir()
+
+        # Test save_begin with interval=2
+        cfg = copy.deepcopy(common_cfg)
+        cfg.default_hooks.checkpoint.interval = 2
+        cfg.default_hooks.checkpoint.save_begin = 6
+        runner = self.build_runner(cfg)
+        runner.train()
+        for i in range(6):
+            self.assertFalse(
+                osp.isfile(osp.join(cfg.work_dir, f'{training_type}_{i}.pth')))
+        for i in range(6, 11):
+            if i % 2:
```

Comment:
Refer to https://google.github.io/styleguide/pyguide.html#214-truefalse-evaluations
```suggestion
            if i % 2 == 1:
```